<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="open">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="num">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="baddata">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>120</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>未开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="usetime">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>读取数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>完整数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>生成折线图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="track">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>显示轨迹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="height">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>显示高程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示碾压遍数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>变异CMV数据筛选</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>导出完整数据</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
